### PR TITLE
Feat teasers and cols for homepage

### PR DESCRIPTION
**Pull request by svenlivingdocs**
- Added new markup and CSS for Cols shortcode
- Added new markup and CSS for Illu-Teaser shortcode
- Added CSS for Flag element
- Updated home page with new teasers

## Expected Result
image

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -20,23 +20,30 @@ If you are already in contact with us and have access to our repositories we rec
...
-{{< img src="illustration.png" alt="Livingdocs in a Box" >}}
-
+## Discover our technology the way you like
+
+{{< cols >}}
+  {{< illu-teaser
+    image="illu-referrence.png"
+    flag="Reference Docs"
+    title="Look up all specifications of our interfaces"
+    link="/reference-docs/public-api/"
```
Comment (DaRaFF) [inline content/_index.md:30]: I would link here to /reference-docs/

File: content/_index.md [hunk 2]
```diff
@@ -20,23 +20,30 @@ If you are already in contact with us and have access to our repositories we rec
...
+  >}}
+
+  {{< illu-teaser
+    image="illu-guides.png"
+    flag="Guides"
+    title="If you would like to get a specific task done"
+    link="/guides/setup/"
```
Comment (DaRaFF) [inline content/_index.md:37]: I would link here to /guides/

File: content/_index.md [hunk 3]
```diff
@@ -20,23 +20,30 @@ If you are already in contact with us and have access to our repositories we rec
...
+  >}}
+
+  {{< illu-teaser
+    image="illu-operations.png"
+    flag="Operations"
+    title="To see common setups, cloud deployments, required services and monitoring"
+    link="/operations/releases/"
```
Comment (DaRaFF) [inline content/_index.md:44]: I would link here to /operations/

**Review by DaRaFF — APPROVED** [review]
Looks super nice 👍 
Please adapt the links as proposed in the comments.
Have you already a plan with the learning section?

**Comment by svenlivingdocs** [comment]
> Looks super nice 👍 Please adapt the links as proposed in the comments. Have you already a plan with the learning section?

Will do, thanks!
When I spoke to Lukas he suggested a fullwidth component for the learning section. Haven't had the time to get to that.
I'll wait with merging for a bit to give Lukas a chance to raise his opinion.

**Comment by peyerluk** [comment]
❤️

**Review by peyerluk — APPROVED** [review]

**Pull request merged**